plutus-ledger-api/src/v3/mod.rs

```diff
@@ -0,0 +1,15 @@
+//! Plutus types and utilities for Plutus V3
+//!
+//! Types and utilities unchanged in the new version are re-exported from the v2 module.
+pub mod ratio;
+pub mod transaction;
+
+// Inherited from v2
+pub use crate::v2::address;
+pub use crate::v2::assoc_map;
+pub use crate::v2::crypto;
+pub use crate::v2::datum;
+pub use crate::v2::interval;
+pub use crate::v2::redeemer;
+pub use crate::v2::script;
+pub use crate::v2::value;
```

Comment:
Oh we have to reexport `TransactionInput` from v1 and `TransactionOutput` from v2 in the `transaction` module